examples/udpblaster/udpblaster.h

```diff
@@ -40,8 +40,6 @@
  * Included Files
  ****************************************************************************/
 
-#include "config.h"
```

Comment:
@xiaoxiang781216 I'm not sure if this is not needed.  My recollection is we need this when running from the host.  The Makefile generates this "config.h".

Reply:
Yes, you are right. I revert the change.